### PR TITLE
Fix type issues

### LINKED ISSUE #115: Issue with types for probeForRoute, decodePaymentRequest and getFeeRates
The types of routes in `probeForRoute` args does not match result of `decodePaymentRequest` so we cannot pass values from decodePaymentRequest to probeForRoute directly without type converstion, typescript will fail to compile.

https://github.com/alexbosworth/lightning/blob/master/lnd_methods/offchain/probe_for_route.d.ts#L49
https://github.com/alexbosworth/lightning/blob/master/lnd_methods/offchain/decode_payment_request.d.ts#L40

Also `tokens` seems to be optional if you pass `mtokens` in the actual code validate function, but it is a required type.

https://github.com/alexbosworth/lightning/blob/master/lnd_methods/offchain/probe_for_route.d.ts#L64
https://github.com/alexbosworth/lightning/blob/master/lnd_methods/offchain/probe_for_route.js#L108

getFeeRates is missing fee_rate in the result.
https://github.com/alexbosworth/lightning/blob/master/lnd_methods/offchain/get_fee_rates.d.ts#L4

### PR DESCRIPTION
**Pull request by bkiac**
Closes #115

- Fix missing fee rate from fee rates result
- Fix issue with tokens being always required in probe
	- now passing either `tokens` or `mtokens` is required	
- Exported route edge type from `decode_payment_request` and extended it in `probe_for_route` so the types will always match

### CODE REVIEW

**Comment by alexbosworth** [comment]
@niteshbalusu11 is this looking like what you need on types?

File: lnd_methods/offchain/probe_for_route.d.ts [hunk 1]
```diff
@@ -1,70 +1,78 @@
...
+    outgoing_channel?: string;
+    /** Time to Spend On A Path Milliseconds Number */
+    path_timeout_ms?: number;
+    /** Payment Identifier Hex String */
+    payment?: string;
+    /** Probe Timeout Milliseconds Number */
+    probe_timeout_ms?: number;
+    routes?: {
+      /** Base Routing Fee In Millitokens Number */
+      base_fee_mtokens?: number;
```
Comment (niteshbalusu11) [inline lnd_methods/offchain/probe_for_route.d.ts:51]: shouldn't this be a string because decodePaymentRequest returns a string?

Comment (bkiac) [reply lnd_methods/offchain/probe_for_route.d.ts]: fixed in [`005fa6f` (#119)](https://github.com/alexbosworth/lightning/pull/119/commits/005fa6f9d4c9cd72a9db5a4219934f5be0860295)

**Comment by niteshbalusu11** [comment]
Looks good now, thanks!

**Comment by niteshbalusu11** [comment]
@alexbosworth can you merge this when you have a chance?

**Pull request merged**